appveyor-gn.yml

```diff
@@ -26,7 +26,6 @@ build_script:
       gclient config
       --name "src\electron"
       --unmanaged
-      --cache-dir "${GIT_CACHE_PATH}"
```

Comment:
We still need this.

Reply:
It's picked up by the environment variable; the command line arg is redundant